src/features/languages/generated/client/client.json

```diff
@@ -1574,7 +1583,7 @@
         "review.form.section.review.title": "Review",
         "review.formData.header": "{isDraft, select, true {Check responses with the informant before sending for review} other {Review the answers with the supporting documents}}",
         "review.header.subject.subjectWithoutName": "{eventType, select, birth {Birth} other {Death}} declaration",
-        "review.header.subject.subjectWitName": "{eventType, select, birth {Birth} other {Death}} declaration for {name}",
+        "review.header.subject.subjectWitName": "{eventType, select, birth {Birth} other {Death}} Declaration for {name}",
```

Comment:
Why do we need to capitalise this @sadmananik

Reply:
Fixed it.